hitl_breakpoint_example.py

```diff
@@ -0,0 +1,183 @@
+# SPDX-FileCopyrightText: 2022-present deepset GmbH <info@deepset.ai>
```

Comment:
This is the example script to show how to run the `BreakpointConfirmationStrategy` which uses Breakpoints and snapshots to pause and resume an Agent.